### PR TITLE
Class 02

### PR DESCRIPTION
**Pull request by austriker27**

### CODE REVIEW

**Comment by austriker27** [comment]
I worked on this for probably 5 hours over the last 2 days. 
the whole thing is challenging including getting the projects to show up via jquery.

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    "no-template-curly-in-string": "error",
+    "no-console": "off",
+    "no-undefined": "off",
+    "indent": ["error", 2],
+    "quotes": ["warn", "single", {"allowTemplateLiterals": true}],
+    "no-multi-spaces": ["warn", {"exceptions": { "VariableDeclarator": true }}],
+    "no-trailing-spaces": "warn",
+    "new-cap": "warn",
+    "no-redeclare": ["error", { "builtinGlobals": true }],
+    "no-unused-vars": "off"
```
Comment (musflood) [inline .eslintrc:33]: You should add a semi-colon rule to this linter:
```
"semicolon": [true, "always"]
```

File: css/base.css [hunk 2]
```diff
@@ -31,21 +26,29 @@
   padding: 2% 2% 2% 2%;
 }
 
+.socialMediaBtns a {
```
Comment (musflood) [inline css/base.css:29]: There should only be element styling in `base.css`, all styling for smaller pieces, like for classes and ids, should be in `modules.css`.

File: .eslintrc (hunk 1 above)
Comment (austriker27) [reply .eslintrc:33]: I added it but now my Atom gives me constant error messages. Is it required?

Comment (musflood) [reply .eslintrc:33]: Sorry, I got that from the docs for a linter, but apparently it was incorrect. The proper syntax for the semi-colon rule is:
```
"semi": ["error", "always"]
```

**Pull request merged**